src/atoms/Icon/icons.module.scss

```diff
@@ -0,0 +1,19 @@
+@import 'src/assets/stylesheets/global';
+
+.icon-wrapper {
+  width: 100%;
```

Comment:
@cisacke You're right about the duplicates -- I'll add new sizes in the navigator, but the idea is to basically set a standard `ru(1)` sq dimension here that can be resized by whatever component uses the icon. FYI this was rebuilt because SVGs can proportionally fill there containers.